### ISSUE TITLE
Export to PPTX doesn't end the loading animation

### ISSUE THREAD

**Issue by Trekky12** [sozi-projects/Sozi#500]
### Summary of your problem or feature request

With the new Sozi pre-release 20.10 the export to PPTX is not stopping the loading animation.

### Steps to reproduce the problem
* Open sozi presentation
* Export to PPTX

### Expected behavior

Presentation is exported and loading animation is stopped.

### Observed behavior

Loading animation is not stopping.

### Hints and solutions (optional)

Logfile:
```
fs.js:752 Uncaught (in promise) Error: ENOTEMPTY: directory not empty, rmdir 'C:\Users\Trekky12\AppData\Local\Temp\tmp-11512-9NUMLz3AsDdD'
    at Object.rmdirSync (fs.js:752)
    at rmkidsSync (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\rimraf\rimraf.js:349)
    at rmdirSync (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\rimraf\rimraf.js:329)
    at rimrafSync (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\rimraf\rimraf.js:301)
    at Object._cleanupCallback [as removeCallback] (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\tmp\lib\tmp.js:356)
    at officegen.<anonymous> (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\js\exporter\exporter.js:426)
    at officegen.emit (events.js:223)
    at generateNextResource (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\officegen\lib\core\index.js:503)
    at officegen.generate (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\node_modules\officegen\lib\core\index.js:507)
    at EventEmitter.<anonymous> (C:\Users\Trekky12\Programme\Sozi\Sozi-20.10.05-1601933294-windows-x64\resources\app\js\exporter\exporter.js:464)
```

**Comment by aumouvantsillage** [comment]
Thanks for reporting.

The PPTX exporter creates a temporary folder where it stores a sequence of image files.
When the PPTX document is complete, the temporary folder and its content should  be deleted entirely.

In your case, the delete operation is refused because the folder is not empty, which is unexpected.
It works fine for me on Ubuntu and I remember making it work on Windows 10 as well.
However, my own attempts tonight produced another error :(

It would help if you could answer these questions:

* Does it happen on every attempt?
* Do you get a correct PPTX file in spite of the error?
* When the error happens, do you have another application (such as a file explorer window) watching the content of the temporary folder while the export is in progress?

In all cases, I will modify the exporter so that it ignores this error when it happens.

**Comment by Trekky12** [comment]
Thank you for your response.

* Yes this happens on every export attempt
* Yes the PPTX is correct despite the error message
* Actually the folder is empty when I try to monitor it. Is it possible that the deletion of the temporary files in the folder is "slower" and the delete folder command is processed before each temporary file is actually deleted?

**Comment by aumouvantsillage** [comment]
> Actually the folder is empty when I try to monitor it

My understanding is that if you keep the folder open in a file explorer, Windows will prevent it from being deleted even if it is empty.
The error message is misleading in this case.

**Comment by Trekky12** [comment]
This is indeed right and so I just watched in the parent folder if the folder is deleted. After the error message I checked the folder and it was empty.

**Comment by aumouvantsillage** [comment]
I will consider this issue fixed in Sozi 20.11.
I cannot guarantee that the folder is deleted, but at least it will not cause an error or hang the export process.

As far as I can tell, the [tmp](https://github.com/raszi/node-tmp) package is configured correctly in Sozi and is expected to delete temporary folders properly. If there is still a bug, I will report it to the package's issue page.
